WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1601,6 +1601,16 @@ CreateContactGroupDialogTable   = @FullWidth@
 CreateContactGroupDialogCell    = padding:0 0 8px 0;
 CreateContactGroupDialogInput   = @FullWidth@
 
+#######################################################################
+#   Grant Rights Dialog
+#######################################################################
+
+GrantRightsDialogOuterContainer     = background-color:@lighten(SecondaryBrandingColor,94)@;
```

Comment:
Agreed, we can have discussion with UX and should make all modals consistent. In that case we should revisit this and all modals to use common properties.